vnext/PropertySheets/ReactPackageDirectories.props

```diff
@@ -19,6 +19,8 @@
     <YogaDir Condition="'$(YogaDir)' == ''">$(ReactNativeDir)\ReactCommon\yoga</YogaDir>
   </PropertyGroup>
 
+  <Import Project="$(ReactNativeWindowsDir)\PropertySheets\ReactPatches.targets" />
```

Comment:
>Import [](start = 3, length = 6)

It seems to be a wrong place to add imports. Generally we should avoid adding targets files into .props files.
Would it be possible to add to some other shared .targets file?

Reply:
+1

If the patches are exclusive to the ReactNative (Facebook) code base, this import would be better suited for `PropertySheets\React.Cpp.props`.

Reply:
Right, it sounds as a better place. This .props file should be about directories. Even the PATCH_RN property is better to be moved to RN specific file.

---
In reply to: [369183249](https://github.com/microsoft/react-native-windows/pull/3912#discussion_r369183249) [](ancestors = 369183249)

Reply:
The motivation behind grouping with directory structure is so that consumers of ReactNativeDir will be guaranteed to get something that works. We will still avoid patching for new projects generated with react-native-cli using Microsoft.ReactNative, since it will not import the property sheet.

What about renaming ReactPackageDirectories.props to ReactPackageDirectories.targets? This is going to end up being a breaking change no matter what we do if we keep this in a targets file, but at least the error should be a lot clearer if someone sees the property sheet is missing. @jonthysell I would also appreciate it if you had any contacts for the affected teams, so we could give them a heads up.

Reply:
Re: PATCH_RN. We're doing this here for now since that's where logic was to determine if we're OSS_RN. I can definitely move it to React.Cpp.props once that's gone.